mender-auth/main.cpp

```diff
@@ -45,5 +57,14 @@ int main(int argc, char *argv[]) {
 		return 1;
 	}
 
+	char const *int_text = argc > 1 ? argv[1] : "42";
+
+	optional<int> oi = to_int(int_text);
+	if (oi) {
+		std::cout << "'" << oi.value() << "' is " << *oi;
+	} else {
+		std::cout << "'" << int_text << "' isn't a number";
+	}
```

Comment:
Please don't pollute this code. :wink:

Reply:
Sure, just wanted to leave an example of how to use it, and that it works.

I'll remove